resources/charts/fork-observer/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+Thank you for installing {{ include "bitcoincore.fullname" . }}.
```

Comment:
this might fail? should reference `fork-observer.fullname` iirc

Reply:
Yeah I didn't do the templates bit. Might ask @m3dwards very nicely for help here when he's back around 👅